src/status_im/contexts/wallet/send/input_amount/view.cljs

```diff
@@ -188,6 +216,9 @@
          (fn []
            (set-input-state #(controlled-input/set-upper-limit % current-limit)))
          [current-limit])
+        (rn/use-effect
+         #(when input-error (debounce/clear-all))
+         [input-error])
```

Comment:
This is a bit out of scope, we can move it to another PR, but I think it is fine to leave it here as it includes many minor fixed for the input.

Let's say user wants to send SNT, the balance is 10 SNT.
Then the user starts typing "1", causing an event to fetch routes is debounced to dispatch in 2 seconds.
Then the user types another "1", now the input shows "11". But the user has only 10 SNT, so the input enters in an error state.
Because the first event was already in the dispatch queue, routes will be fetched with amount of "1" even if the user continued typing an invalid amount.

So, calling `debounce/clear-all` fixes that.